.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - actions
```

Comment:
TODO: Remove this before merging, but leave it for now so I actually have a test case.